tests/distributions/test_transform.py

```diff
@@ -44,10 +44,10 @@
 
 # some transforms (stick breaking) require addition of small slack in order to be numerically
 # stable. The minimal addable slack for float32 is higher thus we need to be less strict
-tol = 1e-7 if pytensor.config.floatX == "float64" else 1e-6
+tol = 1e-7 if pytensor.config.floatX == "float64" else 1e-5
```

Comment:
just in case, float32 was not checked in the CI so the previous tolerance was not taken in account